pom.xml

```diff
@@ -239,6 +239,12 @@
 				<scope>import</scope>
 			</dependency>
 
+            <!-- Commons-codec dependency: solve the issue of making clean install-->
+            <dependency>
+                <groupId>commons-codec</groupId>
```

Comment:
Many thanks for adding this dependency and resolving the [issue between jets3t 0.9.4 compatibility version](https://github.com/SANSA-Stack/SANSA-Template-Maven-Spark/commit/be349be1dbea541868fc94da0972e4ee25bc1e9b).